gym/lib/gym/generators/build_command_generator.rb

```diff
@@ -40,6 +40,7 @@ def options
         options << "-archivePath #{archive_path.shellescape}"
         options << "-derivedDataPath '#{config[:derived_data_path]}'" if config[:derived_data_path]
         options << "-resultBundlePath '#{result_bundle_path}'" if config[:result_bundle]
+        options << "OTHER_SWIFT_FLAGS='-Xfrontend -debug-time-function-bodies' | grep .[0-9]ms | grep -v ^0.[0-9]ms | sort -nr > culprits.txt '#{config[:analyze_build_time]}'" if config[:analyze_build_time]
```

Comment:
Modified to `OTHER_SWIFT_FLAGS=\"\$(inherited) -Xfrontend -debug-time-function-bodies\"`